deepchecks/vision/checks/train_test_validation/image_dataset_drift.py

```diff
@@ -141,3 +142,26 @@ def compute(self, context: Context) -> CheckResult:
             displays.insert(0, headnote)
 
         return CheckResult(value=values_dict, display=displays, header='Image Dataset Drift')
+
+    def add_condition_drift_score_less_than(self, threshold: float = 0.1):
+        """
+        Add condition - require drift score to be less than the threshold.
+
+        The drift score used here is the domain_classifier_drift_Score attribute of the check result.
+        Parameters
```

Comment:
you might need new line before "Parameters" for it to compile correctly, not sure

Reply:
You are correct. @TheSolY you need a new line before the Parameters section

Reply:
Weird that Pylint didn't catch that